.npmignore

```diff
@@ -0,0 +1,23 @@
+# Logs
+logs
+*.log
+
+# Runtime data
+pids
+*.pid
+*.seed
+
+# node-waf configuration
+.lock-wscript
+
+# Dependency directory
+# https://www.npmjs.org/doc/misc/npm-faq.html#should-i-check-my-node_modules-folder-into-git
+node_modules
+*.un~
+yarn.lock
+package-lock.json
+source
+.nvmrc
+.hound.yml
+.eslintrc.yml
+.eslintignore
```

Comment:
Or this.

Reply:
I think so. Like you mentioned, this is the ignore file for distribution on NPM only. The end result is intended to be that the package file (required for modules) and built file (the compiled Gatsby plugin) are the only things shipped through NPM (mimicking the pattern of Gatsby's official plugins. 

Folks work on the Git repo will have everything they need contribute.

From my (admittedly limited) understanding of how NPM works, I think only one of us will be able to push things up to NPM publishing is done through an account.